src/libraries/System.Text.Json/src/System/Text/Json/JsonHelpers.cs

```diff
@@ -570,5 +570,52 @@ static int IndexOfFirstTrailingZero(ReadOnlySpan<byte> span)
                 }
             }
         }
+
+#if !NET
+        // Polyfill for MemoryExtensions.Trim
+        public static ReadOnlySpan<char> Trim(this ReadOnlySpan<char> source)
```

Comment:
Why do these need to be pollyfilled? They are available in netstandard2.0 according to https://apisof.net/catalog/0350bc66847594ddb4d98ceafdb88c14?fx=netstandard2.0

Reply:
You're right, for whatever reason intellisense was complaining when I was working on this. Removed.